docs/architecture/adr-005-social-tips-contract.md

```diff
@@ -0,0 +1,98 @@
+# ADR 005: Social tips contract
+
+## Changelog
+
+- Oct 10, 2022: Initial draft;
+
+## Status
+DRAFT
+
+## Abstract
+This ADR defines the architecture of the Social Tips contract. This contract allow sending tips to a user
+using their know **handle** of a centralized application.  
+The handle can be the @handle of Twitter, the user's nickname on Discord or even an e-mail address. 
```

Comment:
I have a question, why does this contract use centralized apps handles instead of dtag?
If user want to send a tip to the receiver via contract, then he must know the dtag and address, also the application link to the profile. So, honestly, using centralized handles is redundant to me.

Reply:
Just read the jira task description. Can you add the idea sending a user who does not have desmos profile yet?